build.cmd

```diff
@@ -33,6 +33,8 @@ IF %BUILDCMD_DNX_VERSION%=="" (
 	CALL packages\KoreBuild\build\dnvm install %BUILDCMD_DNX_VERSION% -runtime CLR -arch x86 -alias default
 )
 CALL packages\KoreBuild\build\dnvm install default -runtime CoreCLR -arch x86
+CALL packages\KoreBuild\build\dnvm install default -runtime CLR -arch x64
```

Comment:
These lines were present earlier but not sure why they were removed in this commit : https://github.com/aspnet/MusicStore/commit/19f8830a5746f46b6bb86215b0d6cafb9813c73c